runtime/local/src/lib.rs

```diff
@@ -793,9 +805,8 @@ impl pallet_vesting::Config for Runtime {
 parameter_types! {
     pub const DepositPerItem: Balance = deposit(1, 0);
     pub const DepositPerByte: Balance = deposit(0, 1);
-    // The lazy deletion runs inside on_initialize.
-    pub DeletionWeightLimit: Weight = AVERAGE_ON_INITIALIZE_RATIO *
-        RuntimeBlockWeights::get().max_block;
+    // Fallback value if storage deposit limit not set by the user
+    pub const DefaultDepositLimit: Balance = deposit(1024, 1024 * 1024);
```

Comment:
Maybe make it the same as other runtime settings?

Reply:
Right, good catch!